src/nvim/screen.c

```diff
@@ -5278,7 +5278,7 @@ void screen_puts_len(char_u *text, int textlen, int row, int col, int attr)
   int force_redraw_next = FALSE;
   int need_redraw;
 
-  const int l_has_mbyte = has_mbyte;
+  const bool l_has_mbyte = has_mbyte;
```

Comment:
Isn't `has_mbyte` always true, thus allowing any code that relies on `l_has_mbyte` being false to be removed?